protoc-gen-twirp/generator.go

```diff
@@ -1269,8 +1284,8 @@ func (t *twirp) generateServerJSONMethod(service *descriptor.ServiceDescriptorPr
 	t.P(`  reqContent := new(`, t.goTypeName(method.GetInputType()), `)`)
 	t.P(`  unmarshaler := `, t.pkgs["jsonpb"], `.Unmarshaler{AllowUnknownFields: true}`)
 	t.P(`  if err = unmarshaler.Unmarshal(req.Body, reqContent); err != nil {`)
-	t.P(`    s.writeError(ctx, resp, malformedRequestError("the json request could not be decoded"))`)
-	t.P(`    return`)
+	t.P(`       s.handleRequestBodyError(ctx, resp, "the json request could not be decoded", err)`)
+	t.P(`    	return`)
```

Comment:
This still has inconsistent indentation. Notice the first line has 3 spaces and the second has 1 tab. The indentation here is maintained manually because it is inside strings, and then is automatically corrected after the code is generated, but it is still useful to indent properly for readability

Reply:
Fixed